frontend/src/routes/(authenticated)/project/create/+page.ts

```diff
@@ -88,3 +89,60 @@ export async function _projectCodeAvailable(code: string): Promise<boolean> {
   if (!result.ok) throw new Error('Failed to check project code availability');
   return await result.json() as boolean;
 }
+
+export async function _getProjectsByLangCodeAndOrg(input: { orgId: string, langCode: string }): Promise<Readable<ProjectsByLangCodeAndOrgQuery['projectsByLangCodeAndOrg']>> {
+  const client = getClient();
+  //language=GraphQL
+  const results = await client.awaitedQueryStore(fetch,
```

Comment:
it would probably be simpler to use client.query as that just returns a promise, it should make the resulting code simpler since we don't need to deal with stores. The main reason to use a store is for cache updates, but we don't have that here since is just the project creation page.

Reply:
Done in commit 297e8a51.